std/windows/registry.d

```diff
@@ -735,20 +735,21 @@ body
         "Value cannot be set: \"" ~ subKey ~ "\"");
 }
 
-private void regProcessNthKey(HKEY hkey, scope void delegate(scope LONG delegate(DWORD, out string)) dg)
+private void regProcessNthKey(Key key, scope void delegate(scope LONG delegate(DWORD, out string)) dg)
```

Comment:
Anything blocking? ;) - I'm not sure whether decorating the `key` param with `scope` makes a difference.